packages/mui-material/src/MobileStepper/MobileStepper.test.js

```diff
@@ -111,4 +111,22 @@ describe('<MobileStepper />', () => {
     rerender(<MobileStepper {...defaultProps} variant="progress" steps={3} activeStep={2} />);
     expect(screen.getByRole('progressbar').getAttribute('aria-valuenow')).to.equal('100');
   });
+
+  it('should set value correctly when steps is set to 1', () => {
+    const { getByRole } = render(<MobileStepper {...defaultProps} variant="progress" steps={1} />);
+    expect(screen.getByRole('progressbar').getAttribute('aria-valuenow')).to.equal('100');
```

Comment:
Extract duplicate `screen.getByRole` method to one constant:
```js
const ariaValueNow = screen.getByRole('progressbar').getAttribute('aria-valuenow')
```
Same in other places.